upgrading/automated_upgrades.adoc

```diff
@@ -782,25 +782,22 @@ other nodes in their class (`app` versus `infra`), one at a time.
 .. Run `oc get daemonset` to verify the label found under `NODE-SELECTOR`. The
 default value is `storagenode=glusterfs`.
 
-.. Remove the daemonset label from the node:
+.. Add a label (for example, `type=upgrade`) to the node you want to upgrade.
 +
+[NOTE]
+====
+Use the *_upgrade_nodes.yaml_* playbook:
 ----
-$ oc label node <node_name> <daemonset_label>-
+# ansible-playbook -i </path/to/inventory/file> \
+    /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-cluster/upgrades/v3_9/upgrade_nodes.yml \
+    [-e <customized_node_upgrade_variables>] <1>
```

Comment:
I'm not sure how this command is labeling anything. Unless it's done by this -e flag? If so, maybe update the callout to give an example of how a label might be added here?

Reply:
Thank you, Andrea. Yes, I will ask the QE engineer.